torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6080,6 +6080,11 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    dtypesIfCUDA=all_types_and(torch.bool, torch.half, torch.bfloat16),
                    sample_inputs_func=sample_inputs_logit,
                    safe_casts_outputs=True),
+    UnaryUfuncInfo('erfcx',
+                   ref=scipy.special.erfcx if TEST_SCIPY else _NOTHING,
+                   aliases=('erfcx', ),
+                   dtypes=all_types_and(torch.bool),
```

Comment:
This might need to do something like set dtypesIfCUDA to an empty set to avoid running CUDA tests

Reply:
I added a Hack for precisionOverride({'rel': {torch.float32: 3.9e-6}, }) to carry the rtol.

The test_unary_ufuncs.py fails with the default rtol = 1.3e-6 for torch.float32 in some case with large number. Scipy uses double internally while torch.erfcx() uses a templatized dtype.

`AssertionError: False is not true : Scalars failed to compare as equal! Comparing 7.529345559035155e+23 and 7.529357808826142e+23 gives a difference of 1.224979098644775e+18, but the allowed difference with rtol=1.3e-06 and atol=1e-05 is only 9.788165151473984e+17!`